tests/Functional/Controller/Admin/GroupControllerTest.php

```diff
@@ -18,16 +17,11 @@
  */
 class GroupControllerTest extends AuthWebTestCase
 {
-    /**
-     * @var EntityManagerInterface
-     */
-    protected $em;
+    protected EntityManagerInterface $em;
 
-    private $controllerEndpoint = '/admin/group';
+    private String $controllerEndpoint = '/admin/group';
```

Comment:
Strings are a primitive data type in PHP, which are usually in lowercase. Please update this definition (and other) accordingly.

Reply:
My bad, too much rust combined with a sleepy head